packages/story-editor/src/components/panels/design/pageBackground/pageBackground.js

```diff
@@ -239,6 +261,9 @@ function PageBackgroundPanel({ selectedElements, pushUpdate }) {
           />
         </Row>
       )}
+      {failedContrast && (
+        <Warning message={ACCESSIBILITY_COPY.lowContrast.backgroundPanel} />
+      )}
```

Comment:
Thanks for all the updates! Was giving it one more sweep and seeing this:

I added some text that's the same color as an image (black). The contrast check shows correctly on the text but the `page background` also shows the warning even though the text is fine against the background.

May be more of a UX question but I think we should probably show the warning on the element that the text is against rather than the background in this case:

![image](https://user-images.githubusercontent.com/22185279/149589008-252529ba-151c-408a-9139-e1c501221aa7.png)

Reply:
I see what you are saying. In this case the "background" doesn't really matter because the text is on top of the image. Making the image the _actual_ background. Therefore the warning shouldn't appear on the background panel. 

There are going to be a ton of edge cases like this. I am not sure how we are going to handle those. 🤔

Reply:
I'll look into only showing the message here if the failure is cause by background color as opposed to images. Also, the message being here wasn't a requirement so happy to remove it if we decide it's too confusing.

Reply:
Totally - I don't want any scope creep here.

I think either of these would be good:
1. Always the contrast warning from the page background panel
2. Only removing the contrast warning from the page background panel when the text contrast problem isn't from the background